src/backend/browser/sandbox.ts

```diff
@@ -156,18 +166,34 @@ export const api: Sandbox = {
       `Not implemented: showQuestionDialog, request: ${JSON.stringify(obj)}`,
     );
   },
-  showImportFileDialog(/* obj: { title: string } */) {
-    throw new Error(
-      "ブラウザ版では現在ファイルの読み込みをサポートしていません",
-    );
+  async showImportFileDialog(obj) {
+    const fileHandle = await showOpenFilePickerImpl({
+      multiple: false,
+      fileTypes: [
+        obj.extensions
+          ? {
+              description: obj.name || obj.extensions.join(", "),
+              accept: {
+                "application/octet-stream": obj.extensions.map(
+                  (ext) => `.${ext}`,
+                ),
+              },
+            }
+          : {
+              accept: {
+                "application/octet-stream": [".*"],
+              },
+              description: "All files",
```

Comment:
こちらも `obj.name ||` にしとくと良さそう？
```suggestion
              description: obj.name || "All files",
```